lib/parser/instantiation-parser.ts

```diff
@@ -29,7 +29,11 @@ export class InstantiationParser extends ParserBase {
       nextToken = this.consumeToken();
     }
     instantiation.componentName = nextToken;
-    instantiation.lexerToken = nextToken;
+    if (label !== undefined) {
+      instantiation.lexerToken = label;
+    } else {
+      instantiation.lexerToken = nextToken;
+    }
 
```

Comment:
This does not work for instantiations without a label. (For example concurrent procedure calls do not require a label.

Reply:
Also i can sadly not add a comment but OInstantiation also has a label? Why both?